pkgs/development/python-modules/earcut/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  yq-go,
+  lib,
+  numpy,
+  fetchPypi,
+  runCommand,
```

Comment:
runCommand is unused.

Reply:
Another leftover from the test! Late night test I guess...